scripts/publish.sh

```diff
@@ -7,15 +7,17 @@ else
     CMD="echo sbt"
 fi
 
-FULL_VERSIONS="2.10.2 2.10.3 2.10.4 2.10.5 2.10.6 2.10.7 2.11.0 2.11.1 2.11.2 2.11.4 2.11.5 2.11.6 2.11.7 2.11.8 2.11.11 2.11.12 2.12.0 2.12.1 2.12.2 2.12.3 2.12.4 2.12.5 2.12.6 2.13.0-M3"
-BIN_VERSIONS="2.10.7 2.11.12 2.12.6 2.13.0-M3"
+FULL_VERSIONS="2.10.2 2.10.3 2.10.4 2.10.5 2.10.6 2.10.7 2.11.0 2.11.1 2.11.2 2.11.4 2.11.5 2.11.6 2.11.7 2.11.8 2.11.11 2.11.12 2.12.0 2.12.1 2.12.2 2.12.3 2.12.4 2.12.5 2.12.6"
+BIN_VERSIONS="2.10.7 2.11.12 2.12.6"
+LIMITED_VERSIONS="2.13.0-M5"
 CLI_VERSIONS="2.10.7 2.11.12 2.12.6"
 SBT_VERSION="2.10.7"
 SBT1_VERSION="2.12.6"
 SBT1_SBTVERSION="1.0.0"
 
 COMPILER="compiler jUnitPlugin"
 LIBS="library javalibEx ir irJS tools toolsJS jsEnvs jsEnvsTestKit testAdapter stubs testInterface jUnitRuntime"
+LIMITED="compiler jUnitPlugin library javalibEx stubs testInterface jUnitRuntime"
```

Comment:
Could you comment on what limited means? (IIUC it is the bare minimum to compile SJS *for* 2.13 but not *in* 2.13).

Reply:
That's right. It's called Limited here because that's how it was called in `Jenkinsfile` (`limitedCIScalaVersions`). Should I call it `NOTOOLS` or something like that?

Reply:
Done.